project.py

```diff
@@ -81,13 +105,60 @@ def find_board(self, image):
 
         return our_cnt
 
+    def seed_filter(self, image, seed):
```

Comment:
@Foxiy I believe the name of this function should indicate that it is actually a question. For instance I find 

```python
if seed_filter_sufficient(image, seed):
```

```python
if seed_filter_sufficient(image, seed)
```

way more intuitive than

```python
if seed_filter(image, seed):
```